### PR TITLE
Moorpy feedstock

### PR DESCRIPTION
**Pull request by gbarter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gbarter** [comment]
@conda-forge/help-python please review at your convenience.

**Comment by gbarter** [comment]
@mattEhall : Can you please chime in here that you are willing to help maintain this feedstock?

**Comment by mattEhall** [comment]
Chiming in to say I'm willing to help maintain this feedstock.

**Comment by gbarter** [comment]
@conda-forge/help-python : Just coming back to this to see what else remains to be done to push it forward?

**Comment by gbarter** [comment]
@conda-forge/help-python : Looking for a review on this one

File: recipes/moorpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/MoorPy/archive/v{{ version }}.tar.gz
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pCrunch-{{ version }}.tar.gz
+  sha256: d6a408844357f5e8a318875db110047b86a4a3956ea1659f8fbdeb1ec685cfb2
```
Comment (ocefpaf) [inline recipes/moorpy/meta.yaml:11]: We prefer PyPI URLs.

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MoorPy-{{ version }}.tar.gz
  sha256: 959c4cbcf0c7cf1d0ec9acc44efecc888c221fd6e40c0a589aa7292847ad4743
```

File: recipes/moorpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - pyyaml
+    - scipy
+
+test:
+  imports:
+    - moorpy
+  #commands:
+  #  - pytest moorpy
+  #requires:
+  #  - pytest
```
[the remark this answers is not in the document]
Comment (gbarter) [reply recipes/moorpy/meta.yaml]: Removed commented lines that were dangling.

**Comment by gbarter** [comment]
@conda-forge/help-python @ocefpaf - Thank you very much for the prompt response and helpful review comments.  I have accepted all of your suggestions and this PR is therefore ready for another look.

**Pull request merged**